component/alertrules.jsonnet

```diff
@@ -0,0 +1,47 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.system_upgrade_controller;
+
+local alert_rules =
+  kube._Object('monitoring.coreos.com/v1', 'PrometheusRule', 'system-upgrade-controller-alert-rules') {
+    metadata+: {
+      namespace: params.monitoring.prometheusRuleNamespace,
```

Comment:
We generally implement a model where components create rules in their own namespace, and configure the namespace to indicate that the monitoring stack should pick up rules in the namespace (e.g. by labelling the namespace with `SYNMonitoring=main` for Rancher monitoring)